flytekit-examples/src/main/java/org/flyte/examples/SumTask.java

```diff
@@ -29,33 +29,39 @@ public SumTask() {
     super(JacksonSdkType.of(SumInput.class), JacksonSdkType.of(SumOutput.class));
   }
 
-  public static SdkTransform of(SdkBindingData a, SdkBindingData b) {
-    return new SumTask().withInput("a", a).withInput("b", b);
+  public static SdkTransform<SumTask.SumOutput> of(SdkBindingData<Long> a, SdkBindingData<Long> b) {
+    return new SumTask()
+        .withInput("a", a) // this still sucks
```

Comment:
```suggestion
        .withInput("a", a)
```

Even though it sucks :D

Reply:
So honest! 😆